scripts/install_local.py

```diff
@@ -118,14 +118,14 @@ def execute_command(args, cwd=None):
         UI_APP_PATH = UI_PATH + "/app"
         
         print("Updating UI files...")
+        execute_command(["rm", "-rf", "node_modules"], cwd=join(cwd, UI_COMMON_PATH))
         execute_command(["rm", "-rf", ".parcel-cache"], cwd=join(cwd, UI_COMMON_PATH))
         execute_command(["rm", "-rf", "dist"], cwd=join(cwd, UI_COMMON_PATH))
         execute_command(["npm", "install"], cwd=join(cwd, UI_COMMON_PATH))
-        execute_command(["npm", "run", "build"], cwd=join(cwd, UI_COMMON_PATH))
```

Comment:
I'm also not sure if this script will work if we're not actually building the `common` lib.

Reply:
@agiron123

Reply:
@vsreekanti @cw75 ```npm install``` when run inside common actually builds common at the end. This is a redundant command that will save us a couple minutes on build time.